### PR TITLE
Update documentation for condition expressions.

### PR DESCRIPTION
**Pull request by jpinner-lyft**

### CODE REVIEW

File: docs/conditional.rst [hunk 1]
```diff
@@ -1,11 +1,9 @@
 Conditional Operations
 ======================
 
-Some DynamoDB operations (UpdateItem, PutItem, DeleteItem) support the inclusion of conditions. The user can supply a list of conditions to be
-evaluated by DynamoDB before the operation is performed, as well as specifying whether those conditions are
-applied with logical OR (at least one must be true) or logical AND (all must be true). See the `official documentation <http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithItems.html#ConditionalExpressions>`_
-for more details. PynamoDB supports conditionals through keyword arguments, using syntax that is similar to the filter syntax (see :ref:`filtering`).
-Multiple conditions may be supplied, and each value provided will be serialized using the serializer defined for that attribute.
+Some DynamoDB operations (UpdateItem, PutItem, DeleteItem) support the inclusion of conditions. The user can supply a condition to be
+evaluated by DynamoDB before the operation is performed. See the `official documentation <http://http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/WorkingWithItems.html#WorkingWithItems.ConditionalUpdate>`_
```
Comment (garrettheel) [inline docs/conditional.rst]: double `http://`

File: docs/conditional.rst [hunk 2]
```diff
@@ -26,113 +24,95 @@ Suppose that you have defined a `Thread` Model for the examples below.
...
 
-Specifying that the conditions should be applied with AND or OR is achieved through the use of the `conditional_operator` keyword,
-which can be `and` or `or`.
+PynamoDB supports creating condition expressions from attributes using a simple syntax. Attributes can be compared against values.
+Any value provided will be serialized using the serializer defined for that attribute.
 
 .. code-block:: python
 
-    thread_item = Thread('Existing Forum', 'Example Subject')
+    print(Thread.forum_name == 'Some Forum')
```
Comment (garrettheel) [inline docs/conditional.rst]: It's unclear what value the `print` adds here, as there is no interpreter output shown

File: docs/conditional.rst [hunk 3]
```diff
@@ -26,113 +24,95 @@ Suppose that you have defined a `Thread` Model for the examples below.
         views = NumberAttribute(default=0)
 
 
-AND vs. OR
-^^^^^^^^^^
+Condition Expressions
```
Comment (garrettheel) [inline docs/conditional.rst]: I think it'd be really useful to show the conditions being used in a real query, at least once, here

File: docs/conditional.rst [hunk 4]
```diff
@@ -25,114 +23,80 @@ Suppose that you have defined a `Thread` Model for the examples below.
...
+
+Condition Expressions
+^^^^^^^^^^^^^^^^^^^^^
+
+PynamoDB supports creating condition expressions from attributes using a simple syntax.
+Any value provided will be serialized using the serializer defined for that attribute.
+See the `comparison operator and function reference <http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.OperatorsAndFunctions.html>`_
+for more details.
+
+.. csv-table::
```
Comment (jpinner-lyft) [inline docs/conditional.rst]: @lita follow up from https://github.com/pynamodb/PynamoDB/pull/336#discussion_r134660837
expression table

Comment (lita) [reply docs/conditional.rst]: Yes!!!!!

File: docs/conditional.rst [hunk 5]
```diff
@@ -25,114 +23,80 @@ Suppose that you have defined a `Thread` Model for the examples below.
...
-^^^^^^^^^^
-
-Specifying that the conditions should be applied with AND or OR is achieved through the use of the `conditional_operator` keyword,
-which can be `and` or `or`.
+.. _conditions:
+
+Condition Expressions
+^^^^^^^^^^^^^^^^^^^^^
+
+PynamoDB supports creating condition expressions from attributes using a simple syntax.
```
Comment (lita) [inline docs/conditional.rst]: Again, I would state that you are using the Python builtin-operators here.

Comment (jpinner-lyft) [reply docs/conditional.rst]: Do you have a suggested wording? Not all expressions use built-in operators, some are done via methods on the attribute class.

**Pull request merged**